illumos-utils/src/opte/mod.rs

```diff
@@ -66,7 +74,7 @@ impl Gateway {
             ip: subnet
                 .iter()
                 .nth(1)
-                .expect("IP subnet must have at least 1 address"),
+                .expect("IP subnet must have at least 2 addresses"),
```

Comment:
Was this check wrong before?  It seems like it was already trying to return the 2nd element (if I'm understanding `nth()` correctly)

Reply:
This is just updating the message here to be more accurate. You're right in that we're grabbing the second address, hence why the given subnet needs at least 2 addresses to begin with, not 1.

Reply:
Ah, I thought we were adding 2 address support, but you were just being a good citizen and fixing the message.